server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/DefaultMirroringService.java

```diff
@@ -131,12 +131,11 @@ public synchronized void start(CommandExecutor commandExecutor) {
                     }
                 }));
 
-        // Migrate the old mirrors.json to the new format if exists.
         try {
-            new MirroringMigrationService(projectManager, commandExecutor).migrate();
```

Comment:
Note: I understood that it will be mentioned in the release notes clearly that https://github.com/line/centraldogma/pull/880 should be released before this version (0.67.0)

---

Note: I think other parties hosting their own CentralDogma servers may have difficulty upgrading with the recent versions. Ideally, we may want to consider in the future:

1) Add warning logs to check if `hostnamePatterns` is indeed not being used, and guide users to migrate (possibly in the following location)
https://github.com/line/centraldogma/blob/534f6bf8e6e9918d2fcf878612518bd4f25aeee7/server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/credential/AbstractMirrorCredential.java#L92
2) After a couple of releases, remove the `hostnamePatterns` field and data

Reply:
> I understood that it will be mentioned in the release notes clearly that https://github.com/line/centraldogma/pull/880 (0.67.0) should be released before this version

That is correct. 👍 

> Ideally, we may want to consider in the future:

That's a good idea. 👍 
Since the PR hasn't been merged yet, we can still proceed with your suggestion. However, I opted for this approach because:
- This change isn't related to a widely-used client feature.
- We can provide guidance to the server operators, who are fewer in number.
- (Personally) I'd like to wrap up the mirror-related tasks as quickly as possible."